blog/_posts/2018-04-24-kubernetes-application-survey-results-2018.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: blog
+title: 'Kubernetes Application Survey 2018 Results'
+date: '2018-04-24T09:00:00.000-04:00'
+author: mattfarina
+---
+
+Understanding how people use or want to use Kubernetes can help us shape everything from what we build to how we do it. To help us understand how application developers, application operators, and ecosystem tool developers are using and want to use Kubernetes, the Application Definition Working Group recently performed a survey. The survey focused in on these types of user roles and the features and sub-projects owned by the Kubernetes organization. That included kubectl, Dashboard, Minikube, Helm, the Workloads API, etc.
+
+The results are in and the [raw data is now available](https://docs.google.com/spreadsheets/d/12ilRCly2eHKPuicv1P_BD6z__PXAqpiaR-tDYe2eudE/edit?usp=sharing) for everyone.
+
+There is too much data to summarize in a single blog post and we hope people will be able to find useful information by pouring over the data. Here are some of the highlights that caught our attention.
```

Comment:
While I'm happy to include some other data points in the blog post, the primary target audience for this information is those who develop Kubernetes and supporting or ecosystem tools. They are the people who can take the information and use it. This crowd doesn't need to be sold on Kubernetes.

I don't want the post to come across as being about marketing/sales. That wouldn't be genuine for me. When I want to sell a project or idea I show how it would help solve a problem.

The whole goal of the post is to help the people who work on Kubernetes and the ecosystem get the data. Hopefully prior to KubeCon. Sales and case studies aren't going to help them.

Reply:
@mattfarina fair points.